### PR TITLE
improved logic for avoiding errors when data are missing for gage height in vdiagram

### PR DESCRIPTION
**Pull request by mhines-usgs**
more for AQCU-859

### CODE REVIEW

File: tests/testthat/test-vdiagram.R [hunk 1]
```diff
@@ -19,4 +19,20 @@ test_that("vdiagram examples work", {
  expect_is(vdiagram(data, 'html'), 'character')
 })
 
+context("testing vdiagram when gage heights are blank")
+test_that("vdiagram examples work", {
```
Comment (lindsayplatt) [inline tests/testthat/test-vdiagram.R:23]: I think we are trying to add less JSON files to pinpoint what part of the JSON is being tested. See `test-utils-json.R`. I don't think it's necessary to change this one, but just so you know in the future!

**Review by lindsayplatt — APPROVED** [review]

File: tests/testthat/test-vdiagram.R (hunk 1 above)
Comment (mhines-usgs) [reply tests/testthat/test-vdiagram.R:23]: ok, I guess that makes sense, but how should we catch the case where like this task, we're not handling missing data with a specific report and it fails to render?

Comment (lindsayplatt) [reply tests/testthat/test-vdiagram.R:23]: hmmm good point :thinking:

Comment (mhines-usgs) [reply tests/testthat/test-vdiagram.R:23]: I put it into the tech debt list, it's probably worth combing through all the reports and catching these issues before we run into them testing. Thanks!

**Pull request merged**